nowcasting_dataset/data_sources/data_source.py

```diff
@@ -154,26 +148,39 @@ def create_batches(
         Safe to call from worker processes.
 
         Args:
-          spatial_and_temporal_locations_of_each_example: A DataFrame where each row specifies
-            the spatial and temporal location of an example.  The number of rows must be
-            an exact multiple of `batch_size`.
-            Columns are: t0_datetime_UTC, x_center_OSGB, y_center_OSGB.
-          idx_of_first_batch: The batch number of the first batch to create.
-          batch_size: The number of examples per batch.
-          dst_path: The final destination path for the batches.  Must exist.
-          local_temp_path: The local temporary path.  This is only required when dst_path is a
-            cloud storage bucket, so files must first be created on the VM's local disk in temp_path
-            and then uploaded to dst_path every upload_every_n_batches. Must exist. Will be emptied.
-          upload_every_n_batches: Upload the contents of temp_path to dst_path after this number
-            of batches have been created.  If 0 then will write directly to dst_path.
+            spatial_and_temporal_locations_of_each_example (pd.DataFrame): A DataFrame where each
```

Comment:
The text hasn't changed.  All that's changed is the addition of the type in brackets, and deeper indentation to make it easier to read